src/scripts/search-engines/duckduckgo.ts

```diff
@@ -19,14 +19,14 @@ function defaultControlStyle(style?: CSSAttribute): (root: HTMLElement) => void
 export function getSerpHandler(): SerpHandler {
   return handleSerp({
     globalStyle: {
-      '[data-ub-blocked="visible"], [data-ub-blocked="visible"] + .results__sitelink--organics, [data-ub-blocked="visible"] .tile--img__sub':
+      '[data-ub-blocked="visible"], [data-ub-blocked="visible"] + .results__sitelinks--organics, [data-ub-blocked="visible"] .tile--img__sub':
```

Comment:
sorry, typo for `.result__sitelinks--organics`

Reply:
Would you please create a new pull request?

Reply:
opened #125